lib/tests/streamlit/runtime/state/session_state_test.py

```diff
@@ -701,16 +701,19 @@ def nested():
 
 
 @given(state=stst.session_state())
+@settings(deadline=None)
```

Comment:
These tests were taking longer than the proposed 200ms as I added the query_params dictionary so removed the deadline here.

Reply:
Seems pretty extreme to go from having a deadline by default on this test to removing it entirely. Do we know why adding the `query_params` dictionary made these tests take consistently longer?

It might be fine if they were already borderline timing out and the new dict made them take just a tiny bit longer and thus consistently time out, but something's wrong if they're taking substantially longer since dict initialization shouldn't take much time

Reply:
Hm I didn't change anything to `session_state_test.py` and the changes in `session_state.py` seem innocuous so I'm not sure why it's timing out. I've also seen these tests time out occasionally in [other random PRs](https://github.com/streamlit/streamlit/actions/runs/6670711076/job/18131217320) so I'm pretty sure it's unrelated to my PR as far as I can tell. 

PR code for the one failing: https://github.com/streamlit/streamlit/pull/7398/files#diff-6771df6eeaa1f9783734d816075ab2e867001cdd12d0f71983fee5bccd50f27c

I got these logs so it's somewhat borderlining timing out from what I can tell:
```Unreliable test timings! On an initial run, this test took 253.05ms, which exceeded the deadline of 200.00ms, but on a subsequent run it took 3.81 ms, which did not. If you expect this sort of variability in your test timings, consider turning deadlines off for this test by setting deadline=None.```
```Unreliable test timings! On an initial run, this test took 263.13ms, which exceeded the deadline of 200.00ms, but on a subsequent run it took 9.89 ms, which did not. If you expect this sort of variability in your test timings, consider turning deadlines off for this test by setting deadline=None.```

I'm not sure what is exactly why it's going over but I'm also not sure how long it was taking before 🤔 I propose that we set some deadline of 400ms in order to not have any flakey tests or just remove the deadline imo.